src/WriterImpl.h

```diff
@@ -36,7 +37,8 @@ namespace e57
    class WriterImpl
    {
    public:
-      WriterImpl( const ustring &filePath, const ustring &coordinateMetaData );
+      WriterImpl( const ustring &filePath, const ustring &coordinateMetadata );
```

Comment:
I meant to get rid of this version completely. Sorry I wasn't clearer.

It's internal, so it won't break the API.

I'll plan to remove `Writer::Writer( const ustring &filePath, const ustring &coordinateMetadata )` sometime in the future.

Reply:
That makes sense.